packages/mobile/src/navigator/DrawerTopBar.tsx

```diff
@@ -1,46 +1,60 @@
-import colorsV2 from '@celo/react-components/styles/colors.v2'
-import { elevationShadowStyle } from '@celo/react-components/styles/styles'
+import HamburgerIcon from '@celo/react-components/icons/Hamburger'
+import colors from '@celo/react-components/styles/colors.v2'
 import { useNavigation } from '@react-navigation/native'
 import * as React from 'react'
 import { StyleSheet, TouchableOpacity, View } from 'react-native'
-import Logo from 'src/icons/Logo.v2'
-import Hamburger from 'src/navigator/Hamburger'
+import Animated, { cond, greaterThan } from 'react-native-reanimated'
 
-const DrawerTopBar = () => {
+interface Props {
+  middleElement?: React.ReactNode
+  scrollPosition?: Animated.Value<number>
+}
+
+function DrawerTopBar({ middleElement, scrollPosition }: Props) {
   const navigation = useNavigation()
+  const viewStyle = React.useMemo(
+    () => ({
+      ...styles.container,
+      borderBottomColor: colors.gray2,
+      borderBottomWidth: cond(greaterThan(scrollPosition ?? new Animated.Value(0), 0), 1, 0),
+    }),
+    [scrollPosition]
+  )
 
   return (
-    <View style={styles.container}>
+    <Animated.View style={viewStyle}>
       {/*
       // @ts-ignore Only used in a drawer */}
       <TouchableOpacity style={styles.hamburger} onPress={navigation.toggleDrawer}>
-        <Hamburger />
+        <HamburgerIcon />
       </TouchableOpacity>
-      <View style={styles.logo}>
-        <Logo />
-      </View>
+      {middleElement}
       <View style={styles.spacer} />
-    </View>
+    </Animated.View>
   )
 }
 
+DrawerTopBar.defaultProps = {
```

Comment:
This also isn't being applied and I think it should be. It would be nice if the default would be for the bar to have the wallet icon present and have property we can pass in if we want to hide it or show a custom one.

Think that would be the easiest way to avoid importing the logo in a bunch of different places and sometimes forgetting to do it.

Reply:
I see the wallet logo in the top bar only for Home screen. I might be missing something here, are we using somewhere else?

Reply:
Hmm yeah it looks like you are right. For some reason I thought we wanted it in more places but I guess not!